curriculum/structure/superblocks/full-stack-developer.json

```diff
@@ -882,6 +882,11 @@
       "dashedName": "backend-javascript",
       "comingSoon": true,
       "modules": [
+        {
+          "dashedName": "introduction-to-nodejs",
+          "comingSoon": true,
+          "blocks": ["quiz-node-js-intro"]
+        },
```

Comment:
consider removing this change to avoid conflicts with https://github.com/freeCodeCamp/freeCodeCamp/pull/64283